sjsonnet/src/sjsonnet/Format.scala

```diff
@@ -101,7 +101,9 @@ object Format{
       val cooked0 = formatted.conversion match{
         case '%' => widenRaw(formatted, "%")
         case _ =>
-
+          if  (values.isInstanceOf[Val.Arr] && i >= values.cast[Val.Arr].length) {
```

Comment:
Does this `i` print work correctly? For instance, if I give "%s %s %s %s %s" % ["one"] I would expect "Too few values to format: 5, expected 1" where it seems from the iteration this will fail at i = 1, giving a confusing error message.

Reply:
yeah good catch. may be I can rephrase to "at least i values"?

Reply:
Yeah, that sounds good. I don't think it's worth it to do extra parsing just to make the error report the real size of the collection.